src/Uno.UI/UI/Xaml/FontFamily.cs

```diff
@@ -10,6 +11,10 @@ public FontFamily(string familyName)
 		{
 			Source = familyName;
 
+#if __WASM__
```

Comment:
Should be moved to a `FontFamily.wasm.cs` file and initialized through a partial method.

Reply:
Done!